src/main/java/com/microsoft/sqlserver/jdbc/spark/DataFrameBulkRecord.java

```diff
@@ -82,41 +78,11 @@ public int getScale(int column) {
         return dfColumnMetadata[column-1].getScale();
     }
 
-    @Override
-    public boolean isAutoIncrement(int column) {
```

Comment:
@shivsood do we also need to remove the field isAutoIncrement and the function isAutoIncrement() in ColumnMetadata.java?